sass/components/_modals.scss

```diff
@@ -0,0 +1,209 @@
+@import "../base/_animations.scss";
+
+.modals {
+  margin-top: 10rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+// BUTTON STYLES
+.button {
+  display: flex;
+  width: 100%;
+  max-width: 128rem;
+  align-items: center;
+  justify-content: center;
+  margin: 0.8rem;
+  padding: 0.4rem 0.8rem;
+  cursor: pointer;
+  transition: all 60ms ease-in-out;
+  text-align: center;
+  white-space: nowrap;
+  text-decoration: none;
+  appearance: none;
+  line-height: 1.3;
+  font-weight: 500;
+  font-size: 2rem;
+  text-transform: capitalize;
+  color: $color-high-contrast;
+  background-color: $color-primary;
+  border: 0 none;
+  border-radius: 3px;
+}
+
+.button:hover {
+  transition: all 60ms ease;
+  opacity: 0.85;
+  color: $color-high-contrast;
+}
+
+.button:active {
+  text-decoration: none;
+  transition: all 60ms ease;
+  transform: scale(0.97);
+  opacity: 0.75;
+}
+
+.button:visited {
+  text-decoration: none;
+}
+
+.button.add-product {
+  color: $color-high-contrast;
+  background: $color-primary;
+}
+
+.button.edit {
+  color: $color-high-contrast;
+  background: $color-secondary;
+}
+
+.button.delete {
+  color: $color-high-contrast;
+  background: #e92222;
+}
+
+.button.button-bordered {
+  color: $color-high-contrast;
+  border: 2px solid $color-high-contrast;
+  background: transparent;
+}
+.button.button-bordered:hover,
+.button.button-bordered:active {
+  color: #ff7473;
+  border-color: $color-high-contrast;
+  background: $color-high-contrast;
+}
+
+//  MODAL
+// The Modal (background)
+.modal {
+  display: none;    // Hidden by default
+  position: fixed;  // Stay in place
+  z-index: 1;       // Sit on top
+  left: 0;
+  top: 0;
+  width: 100%;      // Full width
+  height: 100%;     // Full height
+  overflow: hidden; // Enable scroll if needed
+  transition: all 0.5s ease 0.06s;
+}
+
+// Modal Content/Box
+.modal-content {
+  background-color: $color-high-contrast;
+  margin: 15% auto; // 15% from the top and centered
+  padding: 1.2rem;
+  max-width: 30%;   // Could be more or less, depending on screen size
+  visibility: hidden;
+  box-shadow: 2px 2px 10px 0px rgba(100, 107, 120, 0.6);
+  border-radius: 5px;
+}
+
+// Close Button
+.close {
+  color: $color-low-contrast;
+  font-size: 2rem;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+}
+
+.close:before {
+  content: "Close";
+  font-size: 1rem;
+  display: none;
+  text-decoration: none;
+  align-self: center;
+  margin-top: 0.2rem;
+  font-weight: 400;
+}
+
+.close:hover:before {
+  display: initial;
+  color: $color-low-contrast;
+}
+
+.close:hover,
+.close:focus {
+  color: #ff6666;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.close:active,
+.close:before:active {
+  transition: all 60ms ease;
+  transform: scale(0.97);
+}
+
+// modal form layout
+.modal-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-header {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
Nitpick: Is there some specific reason you need `row-reverse`?
Correct me if I'm wrong, but I believe that this might cause issues for screen readers, where they would read the page content based on the markup and then this would cause inconsistency with visual interpretation. 🤔